pkg/pillar/cmd/zedrouter/server.go

```diff
@@ -54,8 +54,10 @@ type openstackHandler struct {
 }
 
 // Provides k3s cluster kubeconfig
-type kubeConfigHandler struct {
-	ctx *zedrouterContext
+type appInstMetaHandler struct {
+	ctx             *zedrouterContext
+	maxResponseLen  int
+	publishDataType types.AppInstMetaDataType
```

Comment:
Can you please adjust the comment?
In general it is not clear to me, what is the main difference between old `/eve/v1/kubeconfig` and new `/eve/v1/app/appCustomStatus`.

Reply:
'appCustomStatus' is more generic. `kubeconfig` is specifically for k3s/k8s workflows. I'd rather not mix those, as theoretically they can co-exist.